src/main/kotlin/org/jitsi/videobridge/octo/config/OctoRtpReceiver.kt

```diff
@@ -60,6 +61,7 @@ import java.util.concurrent.atomic.AtomicBoolean
  */
 class OctoRtpReceiver(
     val streamInformationStore: ReadOnlyStreamInformationStore,
+    val eventHandler: TransceiverEventHandler?,
```

Comment:
Why nullable?  It wasn't for the non-octo classes in JMT.

Reply:
If there is a reason to keep it nullable, I'd add a default value of `null`.

Reply:
The instance created in OctoEndpoints doesn't need an event handler